client_sdk/avalon_client_sdk/quorum/quorum_worker_registry_list_client.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2019 Intel Corporation
```

Comment:
We are in the process of creating one SDK for avalon. We are already has PR#234 for that.